core/templates/pages/topics-and-skills-dashboard-page/modals/create-new-topic-modal.component.ts

```diff
@@ -80,6 +80,10 @@ export class CreateNewTopicModalComponent extends ConfirmOrCancelModal {
     if (!this.newlyCreatedTopic.urlFragment) {
       return;
     }
+    this.newlyCreatedTopic.urlFragment = this.newlyCreatedTopic.urlFragment
+      .trim()
+      .toLowerCase()
+      .replace(/\s+/g, '-');
```

Comment:
@Somnath-Mishra Just to check, what's the logic that's being used in the topic editor for the validation of this field (after the topic is created)? Is the logic here (for when to show an error message etc.) the same as the logic used in the main topic editor (can you point to the code)? Just wanted to make sure we are following a standard pattern throughout.

Thanks!

Reply:
@seanlip 
In my latest commit, I also added the same feature for when a new story is created. However, I am unable to find the main topic editor. Could you guide me on how to locate it?

Reply:
@Somnath-Mishra Please see this [wiki page](https://github.com/oppia/oppia/wiki/How-to-access-Oppia-webpages), which should answer your question. Let us know if you still have trouble locating it. Thanks!